pypdf/_reader.py

```diff
@@ -667,6 +667,75 @@ def indexed_key(k: str, fields: Dict[Any, Any]) -> str:
                     ff[indexed_key(cast(str, value["/T"]), ff)] = value.get("/V")
         return ff
 
+    def get_pages_showing_field(
+        self, field: Union[Field, PdfObject, IndirectObject]
+    ) -> List[PageObject]:
+        """
+        provides list of pages where the field is called
+
+        Args:
+            field: Field Object, PdfObject or IndirectObject referencing a Field
+
+        Returns: List:
+            Empty list : the field has no widgets attached
+                         (either hidden field or ancestor field)
+            single page list : page where the widget is present (most common)
+            multiple page list: field multiple times with multiple kids widgets
+                         (ex:radio buttons, field repeated on multiple pages)
+
+        Note:
+            to get page indexes, use:
+            [ p.page_number for p in pdfdoc.get_pages_showing_field(field) ]
+        """
+
+        def _get_inherit(obj: DictionaryObject, key: str) -> Any:
```

Comment:
Should probably be `_get_inherited` instead.

Reply:
fixed